### PR TITLE
Emailing bug fix

### PR DESCRIPTION
**Pull request by YakMan101**
Turns out the sentiment analyser wasn't configured with a complete docker file, was missing the clean_content.py file.

Also updated couple READMEs.

### CODE REVIEW

File: news_sentiment_analyser/Dockerfile [hunk 1]
```diff
@@ -17,6 +17,7 @@ COPY transform_articles.py .
 COPY load_rds.py .
 COPY database_functions.py .
 COPY pipeline_analysis.py .
+COPY clean_content.py .
```
Comment (megansam5) [inline news_sentiment_analyser/Dockerfile:20]: Well done for adding this in!!

File: dashboard/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+*.env
```
Comment (indiah444) [inline dashboard/.gitignore:1]: Really important to gitignore this, good spot!

Comment (megansam5) [inline dashboard/.gitignore:1]: Essential!

**Review by megansam5 — APPROVED** [review]
Nice small changes!

**Pull request merged**